src/classes/RD2_EnablementService_TEST.cls

```diff
@@ -0,0 +1,446 @@
+    * @param rdId Recurring Donation Id
+    * @param accountId Account Id
+    * @return Opportunity 
+    */
+    private static Opportunity buildOpportunity(Id rdId, Id accountId) {
+        return new TEST_OpportunityBuilder()
+            .withAmount(100)
+            .withAccount(accountId)
+            .withClosedWonStage()
+            .withCloseDate(Date.today())
+            .withRecurringDonation(rdId)
+            .build();
+    }
+
+    /****
+    * @description Returns Opportunities associated with the specified Recurring Donation
+    * @param rdId Recurring Donation Id
+    * @return List<Opportunity>
+    */
+    private static List<Opportunity> getOpportunities(Id rdId) {
+        return getOpportunities(new Set<Id>{ rdId });
+    }
+
+    /****
+    * @description Returns Opportunities associated with specified Recurring Donations
+    * @param rdIds Recurring Donation Ids
+    * @return List<Opportunity>
+    */
+    private static List<Opportunity> getOpportunities(Set<Id> rdIds) {
+        return [
+            SELECT Amount, npe03__Recurring_Donation__c
+            FROM Opportunity
+            WHERE npe03__Recurring_Donation__c IN :rdIds
+        ];
+    }
+
+    /***
+    * @description Enables or disables customizable rollups for testing
+    * @param isEnabled Indicates if the CRLP should be enabled in the custom settings
+    * @param none
+    */
+    private static void setCustomizableRollups(Boolean isEnabled) {
+        UTIL_CustomSettingsFacade.getRollupSettingsForTests(new Customizable_Rollup_Settings__c(
+            Customizable_Rollups_Enabled__c = isEnabled
+        ));
+    }
+
+    /***
+    * @description Stubs PilotEnabled feature parameter enablement
+    * @param isEnabled Indicates if the PilotEnabled should be enabled using a Mock feature management
+    * @return none
+    */
+    private static void setPilotEnabledFeatureParameter(Boolean isEnabled) {
+        UTIL_FeatureEnablement featureEnablement = UTIL_FeatureEnablement.getInstance();
+
+        featureEnablement.featureManagement = (UTIL_FeatureManagement) Test.createStub(
+            UTIL_FeatureManagement.class, new UTIL_FeatureManagement_TEST.Mock()
+        );
+
+        featureEnablement.featureManagement.setPackageBooleanValue(
+            UTIL_FeatureEnablement.FeatureName.PilotEnabled.name(), isEnabled
+        );
+    }
+
+    /***
+    * @description Stubs custom settings configuration to enable enhanced Recurring Donations
+    * @return none
+    */
+    private static void configureRecurringDonations2Enabled() {        
+        UTIL_CustomSettingsFacade.getRecurringDonationsSettingsForTest(
+            new npe03__Recurring_Donations_Settings__c(
+                IsRecurringDonations2Enabled__c = true,
+                npe03__Open_Opportunity_Behavior__c = RD_RecurringDonations.RecurringDonationCloseOptions.Mark_Opportunities_Closed_Lost.name(),
+                npe03__Opportunity_Forecast_Months__c = 12
```

Comment:
This is a field that we're effectively deprecating in RD2. If you do want to set it here, it should be set to 1 though since our plan is to _never_ create more than one future Opp per RD.